### ISSUE TITLE
cuda tensor instead of int in 1.dqn

### ISSUE THREAD

**Issue by garkavem** [higgsfield/RL-Adventure#10]
Hi! Thanks for the great tutorials!
I had an issue with `class DQN(nn.Module)`, in method `act` this thing 
`action  = q_value.max(1)[1].data[0]`
seemed to return some torch cuda tensor, that `env.step` naturally couldn't take as input. 
I replaced it with with
`action  = int(q_value.max(1)[1].data[0].cpu().int().numpy())`
and it works for me.

**Comment by RaviTej310** [comment]
Yeah, I needed to do the same. I also needed to change `losses.append(loss.data[0])` to `losses.append(loss.data.item())`, in the first cell under the heading "Training".
